config/lib/express.js

```diff
@@ -89,6 +90,21 @@ module.exports.initMiddleware = function (app) {
   // Add the cookie parser and flash middleware
   app.use(cookieParser());
   app.use(flash());
+
+  // Authorize JWT
+  app.use(function (req, res, next) {
```

Comment:
So you move the authorize method out of express into it's own authorization "class". Seems a good idea to separate those
+ it makes the file a little shorter, not that it is too big now but it could easily become too big as someone needs to add something more.